proxy/slither.config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "detectors_to_exclude": "uninitialized-local,calls-loop,reentrancy-events,assembly,solc-version,timestamp,unused-return,variable-scope,dead-code,erc20-interface",
+    "detectors_to_exclude": "uninitialized-local,calls-loop,reentrancy-events,assembly,solc-version,timestamp,unused-return,variable-scope,dead-code,erc20-interface,constable-states",
```

Comment:
Why to disable `constable-states`? It's a good optimization.

Reply:
Will return back